templates/pages/admin/rules/engine_preview_criteria.html.twig

```diff
@@ -0,0 +1,74 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2023 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% import 'components/form/fields_macros.html.twig' as fields %}
+{% import 'components/form/basic_inputs_macros.html.twig' as inputs %}
+
+{% if input|length %}
+   <div class="card">
+      <div class="card-header">
+         <h3 class="card-title">{{ _n('Criterion', 'Criteria', get_plural_number()) }}</h3>
+      </div>
+      <div class="card-body d-flex flex-wrap">
+         <form name="testrule_form" id="testrulesengine_form" method="post" action="{{ params.target }}">
+            <div class="col-12 col-xxl12 flex-column">
+               <div class="d-flex flex-row flex-wrap flex-xl-nowrap">
+                  <div class="row flex-row align-items-start flex-grow-1">
+                     <div class="row flex-row">
+                        {% for criterion in input %}
+                           {% set field_content %}
```

Comment:
One last point, as the html here is generated by `Rule::displayCriteriaSelectPattern`, the default case (text input at l2645) will call an `Html::input` with size = 70.
We should completly remove the size attribute to avoid breaking preview UI.

Apart this, all ok 👍🏽